sdk-libs/macros/src/compressible/GUIDE.md

```
@@ -0,0 +1,198 @@
+
+#[account]
+#[derive(Compressible)]
+pub struct UserRecord {
+    pub compression_info: Option<CompressionInfo>,
+    pub owner: Pubkey,
+    pub name: String,
+    pub score: u64,
+}
+```
+
+2. Generate compress/decompress instructions with auto seeds
```

markdownlint-cli2 (0.18.1)

Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)